starlark-rust/vscode/package.json

```diff
@@ -49,59 +46,21 @@
                     "TARGETS.v2",
                     "PACKAGE"
                 ],
-                "configuration": "./syntaxes/starlark.configuration.json"
+                "configuration": "./syntax/starlark.configuration.json"
             }
         ],
         "grammars": [
             {
                 "language": "starlark",
                 "scopeName": "source.starlark",
-                "path": "./syntaxes/starlark.tmLanguage.json"
-            }
-        ],
-        "breakpoints": [
-            {
-                "language": "starlark"
```

Comment:
Why delete the debugger/breakpoint features?

Reply:
Due to lack of implementation within `buck2 lsp` I noticed later that they were under a different command. Didn't had time yet to reimplement them